go.mod

```diff
@@ -4,27 +4,26 @@ go 1.15
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
 	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415 // indirect
 	github.com/google/btree v0.0.0-20160524151835-7d79101e329e // indirect
 	github.com/google/go-cmp v0.4.0
-	github.com/google/gofuzz v1.0.0 // indirect
-	github.com/googleapis/gnostic v0.0.0-20170729233727-0c5108395e2d // indirect
 	github.com/gregjones/httpcache v0.0.0-20170728041850-787624de3eb7 // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pkg/errors v0.9.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/crypto v0.1.0 // indirect
+	golang.org/x/crypto v0.0.0-20180808211826-de0752318171 // indirect
```

Comment:
Problem with this is that you're removing the (old version 0.1.0](https://github.com/che-incubator/kubernetes-image-puller/pull/155/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L20-R17) from Oct 2022 ([latest is 0.11 from Jul 5 2023](https://pkg.go.dev/golang.org/x/crypto?tab=versions)) and going back to a version from 2018. That seems like a bad choice, given this is a crypto package and we just spent 2 weeks fighting with crypto issues for FIPS compliance. I'll have a look today to see if the KIP and KIPO are FIPS-compliant ... if they're not, then we may have to fix them or else state that the imagepuller doesn't work on FIPS clusters.

Reply:
@nickboldt whatever works, please go ahead and update the modules to be compliant with brew builds

Reply:
FYI https://github.com/eclipse/che/issues/22416

Reply:
> That seems like a bad choice, given this is a crypto package and we just spent 2 weeks fighting with crypto issues for FIPS compliance.

this particular commit with version bump was added to the main Jul 27 and was not part of the r FIPS compliance exercise. Feel free to close it or update if needed but it is the very same version of image puller that was packaged for 3.7 / 3.7.1